doc/chapters/integration-guidelines/examples/tc.sh

```diff
@@ -0,0 +1,3 @@
+tc qdisc add dev eth0 root handle 10: htb
+tc class add dev eth0 parent 10: classid 10:1 htb rate 1mbit
+tc filter add dev eth0 parent 10: protocol ip prio 10 handle 1: cgroup
```

Comment:
I would still prefer to have sudo in there, to indicate that the commands should be run as root